packages/fluentui/react-northstar/src/themes/teams/components/Avatar/avatarVariables.ts

```diff
@@ -5,17 +5,39 @@ export interface AvatarVariables {
   avatarBorderWidth: string;
   squareAvatarBorderRadius: string;
   statusBorderColor: string;
-  statusBorderWidth: string;
+  statusBorderWidth: number;
   iconColor: string;
   iconBackgroundColor: string;
+  statusIconSize: string;
+  statusSuccessBackgroundColor: string;
+  statusSuccessTextColor: string;
+  statusInfoBackgroundColor: string;
+  statusInfoTextColor: string;
+  statusWarningBackgroundColor: string;
+  statusWarningTextColor: string;
+  statusErrorBackgroundColor: string;
+  statusErrorTextColor: string;
+  statusDefaultBackgroundColor: string;
+  statusDefaultTextColor: string;
```

Comment:
```suggestion
  statusColor: string;
```

We don't use `default` prefix. It also seems that `text` prefix also redundant there

Reply:
`Text` prefix is still present in other variables

Reply:
resolved